go-controller/pkg/ovn/multihoming_test.go

```diff
@@ -386,13 +386,26 @@ func enableICFeatureConfig() *config.OVNKubernetesFeatureConfig {
 	return featConfig
 }
 
+func enableEgressFirewallConfig() *config.OVNKubernetesFeatureConfig {
```

Comment:
@maiqueb I see such utils are added everywhere across unit test files.. this is just pure duplication over and over just to enable some feature flags, I prefer being explicit instead of having a bunch of nonIC/IC function utils some in multihoming, some in secondary-unit tests etc etc to hide that aspect and I see all these icEnable/disable and same things repeated - please check other legacy unit tests to see how we did it there -> also minimalConfig means nothing from the name itself
this is anti-pattern to what we have in other files, we usually just use the config knobs, booleans are simple enough to be read clearly as is @pperiyasamy can we not have this many inner constructs just to enable 1 flag? what I mean is enableEFW is calling enableIC ?? not intuitive at all which calls minimal config? not intuitive at all - like I need to read all these utils to get what's happening.. can we not just enable it within the IT blocks and simply reset all the configs when we are done in the beforeeach or something?